package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aca-final-project",
-  "homepage": "http://seanphilippi.github.io/ymdb",
+  "homepage": "http://seanphilippi.github.io",
```

Comment:
The browser was adding the extra `ymdb` when looking for files. Removing this removes them from the file path